Dockerfile-base

```diff
@@ -24,19 +23,27 @@ RUN apt-get update && \
 # bug 1301116
 RUN pip install setuptools==26.1.1
 
+# Install the Node.js 6.9.2 LTS release.
+ENV DEB_PKG=nodejs_6.9.2-1nodesource1~jessie1_amd64.deb
+RUN curl -sLO https://deb.nodesource.com/node_6.x/pool/main/n/nodejs/${DEB_PKG}\
+    && dpkg -i ${DEB_PKG} \
+    && rm ${DEB_PKG} \
+    && apt-get install -f
+
 RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
 
 RUN npm install -g \
```

Comment:
Add `node-sass@4.3.0` to the globally installed node packages to accommodate the upcoming change from stylus to sass.